### PR TITLE
Add "City battle" map

### PR DESCRIPTION
**Pull request by JostP**
I made new map, called **London**. I found interesting minecraft world "World of Worlds" (http://www.minecraftmaps.com/city-maps/world-of-worlds, **created by Zeemo**). I converted it to minetest map (with using sofar's program https://forum.minetest.net/viewtopic.php?f=12&t=13709) and made CTF map from it.

This PR is still **WIP** . It will be finished in April, I hope. Hint is temporary.

Although, map haven't been finished yet, I will be glad if you review it and tell me what you like and what not or give me a suggestion :).

Some screenshots of London map:

Zaslonska slika 2019-03-28 15-55-41
Zaslonska slika 2019-03-28 15-56-10
Zaslonska slika 2019-03-28 15-57-03
Zaslonska slika 2019-03-28 15-57-37
Zaslonska slika 2019-03-28 15-58-11
Zaslonska slika 2019-03-28 15-58-50

### CODE REVIEW

**Comment by Thomas--S** [comment]
What about license?

**Comment by JostP** [comment]
Here is Zeemo's comment/replay:

`Zeemo Quoting HighCraftMC:

    Hello i am a server owner can i use your map on my server?


No problems if you give me credit.
In description, in your hub or in video with a view of my map you have to include "From World of Worlds map created by Zeemo". And have fun my friend ;) `

**Comment by Thomas--S** [comment]
I am not a lawyer, but honestly, I'd strongly prefer some sort of commonly accepted license and not just a "you can use this if...".

Apart from that, I appreciate your efforts in providing more and more maps.

**Comment by rubenwardy** [comment]
Given recent incidents, shooting up inside the houses of parliament isn't something I'd be comfortable with. I suggest choosing somewhere with The Tower of London, or maybe london bridge

**Comment by JostP** [comment]
Map is big, so I think it won't be hard to find interesting part without churches, parliaments or controversial things. I will have to change map's name because there is not enough London buildings anymore (except small London bridge).

If anyone is interested in making CTF map from "World of Worlds" map, I can send him/her a whole map and he/she can choose his/her own part.

I will also ask Zeemo about license.

**Comment by JostP** [comment]
**Update: new part of the original map.** Map is still WIP.

TODO:
* [ ] remowe/replace not-supported nodes from different mods (nether, mesecons, quartz, hardenedclay, carpet etc.)
* [ ] fix barrier and bottom of the map.
* [ ] if needed, change flag positions.
* [ ] add more chests.
* [ ] add ladder or stairs to skyscrapers and Big Ben to enable players to climb up and let them see beautiful view.
* [ ] add license (I have already asked him and I am waiting for answer)

Please review it or look at the screenshots and tell me if you find something wrong or controversial.

Some screenshots:
Zaslonska slika 2019-03-30 14-27-06
Zaslonska slika 2019-03-30 14-26-32
Zaslonska slika 2019-03-30 14-27-28
Zaslonska slika 2019-03-30 14-27-52
Zaslonska slika 2019-03-30 14-28-55

File: city_battle.conf [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name = London
+name = City battle
 author = Zeemo (minecraft player) and -sniper-
 hint = From "World of Worlds" map (minecraft map) created by Zeemo.
```
Comment (ClobberXD) [inline city_battle.conf]: I'd really prefer to not misuse the `hint` field for attribution.

Thanks for reminding me about MT-CTF/capturetheflag#381, by the way ;)

Comment (JostP) [reply city_battle.conf]: Hint is temporary, it will be removed when you finished #381.

**Comment by JostP** [comment]
Whenever I try to [post comment](http://www.minecraftmaps.com/city-maps/world-of-worlds) (you don't have to register to post it), comment isn't posted. I have no idea how to ask Zeemo about license.

If I understand right, no license = copyright license (by default) (https://choosealicense.com/no-permission/). We mustn't merge modified map here on github.

But Zeemo allowed using it on private server if we give him a credit. Is possible to add this map **only** to server and don't merge it on github and noone will be able get modified map? Of course, do this only if you think this map is something special. If you don't think this map is something special then we shouldn't add it and I will close PR.

I will be glad, if someone try to ask Zeemo about license. Maybe on his YouTube channel.

**Comment by JostP** [comment]
Whenever I try to [post comment](http://www.minecraftmaps.com/city-maps/world-of-worlds) (you don't have to register to post it), comment isn't posted. I have no idea how to ask Zeemo about license.

If I understand right, no license = copyright license (by default) (https://choosealicense.com/no-permission/). We mustn't merge modified map here on github.

But Zeemo allowed using it on private server if we give him a credit. Is possible to add this map **only** to server and don't merge it on github and noone will be able get modified map? Of course, do this only if you think this map is something special. If you don't think this map is something special then we shouldn't add it and I will close PR.

I will be glad, if someone try to ask Zeemo about license. Maybe on his YouTube channel.

**Comment by JostP** [comment]
Whenever I try to post comment (you don't have to register to post it), comment isn't posted. I have no idea how to ask Zeemo about license.

If I understand right, no license = copyright license (by default) (https://choosealicense.com/no-permission/). We mustn't merge modified map here on github.

But Zeemo allowed using it on private server if we give him a credit. Is possible to add this map only to server and don't merge it on github and noone will be able get modified map? Of course, do this only if you think this map is something special. If you don't think this map is something special then we shouldn't add it and I will close PR.

I will be glad, if someone try to ask Zeemo about license. Maybe on his YouTube channel.

**Comment by Thomas--S** [comment]
I don't think that using this map only on the server is a good idea, as this is basically a form of public distribution too, in my opinion (especially because of local map saving).

I'd prefer a city style map with appropriate licenses (e.g. Karsthafen).

Please note: I am not a lawyer.

**Pull request closed without merging**